mitodl/edx-platform
# Changed remote gradebook auth settings

#### What are the relevant tickets?

Closes #15 
⚠️  REBASED ON mitodl#76 ⚠️ 

#### What's this PR do?

Changes the way we handle remote gradebook auth to make the request secure

#### How should this be manually tested?

- Make sure the `ENABLE_INSTRUCTOR_REMOTE_GRADEBOOK_CONTROLS` feature is set to `true`
- Set `REMOTE_GRADEBOOK['URL']` in env settings (`lms.env.json`)
- Either set 'Remote Gradebook' value in advanced course settings, or set `REMOTE_GRADEBOOK['DEFAULT_NAME']` in env settings
- Set `REMOTE_GRADEBOOK_USER` and `REMOTE_GRADEBOOK_PASSWORD` in auth settings (`lms.auth.json`)
- Navigate to the 'Remote Gradebook' tab and check that everything works
